frontend/src/screens/learner.tsx

```diff
@@ -168,34 +170,59 @@ export const MobileStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies })
 }
 
 export const DesktopStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies }) => {
+
+    const [displayArrows, setDisplayArrows] = useState<boolean>(false)
+    const [isOverflowing, setIsOverflowing] = useState(false);
+    const viewport = useRef<HTMLDivElement>(null);
+
+    useEffect(() => {
+        const checkOverflow = () => {
+            if (viewport.current) {
+                const overflow = viewport.current.scrollWidth > viewport.current.clientWidth;
+                setIsOverflowing(overflow);
```

Comment:
why not just use the `setDisplayArrows` here instead? then you dont need the overFlow state?

Reply:
I think I can simplify this furthur